src/tests/ftest/container/container_check.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+from command_utils import CommandFailure
+from daos_utils import DaosCommand
+from test_utils_pool import TestPool
+from test_utils_container import TestContainer
+from dfuse_utils import Dfuse
+from apricot import TestWithServers
+
+class DfuseContainerCheck(TestWithServers):
+    """Base Dfuse Container check test class.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a DfuseContainerCheck object."""
+        super(DfuseContainerCheck, self).__init__(*args, **kwargs)
+        self.dfuse = None
+        self.pool = None
+        self.container = None
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super(DfuseContainerCheck, self).setUp()
+
+    def tearDown(self):
+        """Tear down each test case."""
+        try:
+            if self.dfuse:
+                self.dfuse.stop()
+        finally:
+            # Stop the servers and agents
+            super(DfuseContainerCheck, self).tearDown()
+
+    def create_pool(self):
+        """Create a TestPool object to use with ior."""
+        # Get the pool params
+        self.pool = TestPool(
+            self.context, dmg_command=self.get_dmg_command())
+        self.pool.get_params(self)
+
+        # Create a pool
+        self.pool.create()
+
+    def start_dfuse(self):
+        """Create a DfuseCommand object to start dfuse.
+        """
+
+        # Get Dfuse params
+        self.dfuse = Dfuse(self.hostlist_clients, self.tmp)
+        self.dfuse.get_params(self)
+
+        # update dfuse params
+        self.dfuse.set_dfuse_params(self.pool)
+        self.dfuse.set_dfuse_cont_param(self.container)
+        self.dfuse.set_dfuse_exports(self.server_managers[0], self.client_log)
+
+        try:
+            # start dfuse
+            self.dfuse.run(False)
```

Comment:
Why would you set check=False ? If the dfuse has not started right away, there is a 5 second and 25 second wait time to try a couple of times. Is it because you have only one client ?